testing/Google.Api.Gax.Testing/FakeScheduler.cs

```diff
@@ -225,6 +225,11 @@ private Task StartLoopAsync(DateTime simulatedTimeout)
                                 return;
                             }
                         }
+                        // Give some time for tasks to execute.
```

Comment:
So is this the fix that we've wanted for ages but haven't been able to work out, that basically makes FakeScheduler more reliable in general?

I must admit I can't remember enough about it to understand why that loop would work, but I'm sure you're right :)

Reply:
I'm not sure if this is *the* one true fix; but it certainly was required to make some of the new tests pass :)